docs/getting-started/01-overview.md

```diff
@@ -0,0 +1,81 @@
+
+2. Download the `kubeconfig` file with the cluster configuration:
+
+a. Access the Console UI of your Kyma cluster.
+
+b. Click the user icon in the upper right corner.
+
+c. Select **Get Kubeconfig** from the drop-down menu to download the configuration file to a selected location on your machine.
+
+d. Open a terminal window.
+
+e. Export the **KUBECONFIG** environment variable to point to the downloaded `kubeconfig`. Run this command:
+
+   ```bash
+   export KUBECONFIG={KUBECONFIG_FILE_PATH}
+   ```
+
+   >**NOTE:** Drag and drop the `kubeconfig` file in the terminal to easily add the path of the file to the `export KUBECONFIG` command you run.
+
+f. Run `kubectl cluster-info` to check if you are connected to the correct cluster.
+
+## Main actors
+
+Let's introduce the main actors that will lead us through the guides. These are our in-house examples, mock applications, and experimental services:
+
+- [`orders-service`](https://github.com/kyma-project/examples/tree/master/orders-service) is a sample microservice written in Go. It can expose HTTP endpoints that can be used to create and read basic order JSON entities. The service can run with either an in-memory database that is enabled by default or an external Redis database. You will see on its example how you can use Kyma to deploy your own microservice, expose it on HTTP endpoints to make it available for external services, and then bind it to an external database service (Redis).
+
+- [`orders-function`](https://github.com/kyma-project/examples/tree/master/orders-service/deployment/function.yaml) that is an equivalent of `orders-service` with the ability to retrieve order records and save data in it. Like the microservice, the Function can run with either an in-memory database or a Redis instance.
+
+- [Redis addon](https://github.com/kyma-project/addons/tree/master/addons/redis-0.0.3) is basically a bundle of two Redis services available in two plans: `micro` and `enterprise`. You will connect it to Kyma through Helm Broker and expose it in the Kyma cluster as an addon under Service Catalog. The Redis service represents an open-source, in-memory data structure store, used as a database, cache and message broker. For the purpose of these guides, you will use the `micro` plan with in-memory storage to demonstrate how it can replace the default memory of the microservice.
+
+- [Commerce mock](https://github.com/SAP-samples/xf-addons/tree/master/addons/commerce-mock-0.1.0) is to act as a sample external and monolithic application which we want to extend with Kyma. It is based on the [Varkes](https://github.com/kyma-incubator/varkes) project and is also available in the form of an addon. It will simulate how you can pair an external application with Kyma and expose the application's APIs and events. In these guides, you will use its `order.deliverysent.v1` event type to trigger both the microservice and the Function.
```

Comment:
```suggestion
- [Commerce mock](https://github.com/SAP-samples/xf-addons/tree/master/addons/commerce-mock-0.1.0) is to act as a sample external monolithic application which we want to extend with Kyma. It is based on the [Varkes](https://github.com/kyma-incubator/varkes) project and is also available in the form of an addon. It will simulate how you can pair an external application with Kyma and expose the application's APIs and events. In these guides, you will use its `order.deliverysent.v1` event type to trigger both the microservice and the Function.
```

Reply:
I changed it into `is to act as an example of an external monolithic application` as I thought 3 adj in a row is way too much.

Reply:
I like what you did here :) Nice 👍